README.md

```diff
@@ -1,15 +1,46 @@
+[![Review Assignment Due Date](https://classroom.github.com/assets/deadline-readme-button-24ddc0f5d75046c5622901739e7c5dd533143b0c8e959d652212380cedb1ea36.svg)](https://classroom.github.com/a/FgMJElkj)
 # Theory vs. Practice
 
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
 
+  1. The asymptotic analysis may be leading because of the constants that were
+     were dropped when solving for the complexity. They actually affect the runtime
+     of the algorithm (specific timing), where as the complexity simply explains
+     how runtime should look.
+
+  2. The arrangement of the input can also make the asymptotic analysis differ.
+     The average case complexity covers a wide range of possibilities, so the
+     specific input given can act differently during runtime because average case
+     covers what the average case is deemed.
```

Comment:
Don't understand what you mean here. If it's very different, it wouldn't be the average case anymore.